autogluon/scheduler/remote/remote.py

```diff
@@ -68,6 +68,9 @@ def __init__(self, remote_ip=None, port=None, local=False, ssh_username=None,
         else:
             remote_addr = (remote_ip + ':{}'.format(port))
             self.service = start_service(remote_ip, port)
+            _set_global_remote_service(self.service)
+            import time
+            time.sleep(10)
```

Comment:
Can this be a lower value? What impact will this have on repeated HPO runs for many different models like in Tabular?

Reply:
1). This does not impact single machine training. 

2). The 10 seconds overhead is introduced only once at the starting of the remote nodes, which is usually acceptable.  There won't be any overhead during the job scheduling.

Reply:
> Can this be a lower value? What impact will this have on repeated HPO runs for many different models like in Tabular?

There is no overhead for repeated run.